revel/run.go

```diff
@@ -89,7 +89,12 @@ func parseRunArgs(args []string) *RunArgs {
 		// 1. revel run [import-path]
 		// 2. revel run [port]
 		// 3. revel run [run-mode]
-		if _, err := build.Import(args[0], "", build.FindOnly); err == nil {
+		_, err := build.Import(args[0], "", build.FindOnly)
+		if err != nil {
+			revel.WARN.Printf("Unable to run using an import path, assuming import path is working directory %s %s", args[0], err.Error())
```

Comment:
This occurs when you do not specify the application to run. Currently the command silently fails and assumes that the working directory is what you want to launch. This typically can cause really misleading error messages down the road if that was not the case. With this catch at least you have some idea what revel did